### DISCUSSION TITLE
API getting error 28 SSL in ipv6

### DISCUSSION THREAD

**Discussion by tootai** [python-telegram-bot/python-telegram-bot#4165] [Q&A]
Hi list,

I use Telegram bot on a Debian12 ipv6 server running librenms and Transport being Telegram. I also wrote a script to send messages to a group. Since those last days I didn't receive any alert from Librenms and I discover that curl fail with error 28 SSL connection timeout when testing from Librenms. An error occurs too after 2 minutes when I run curl from command line like

curl https://api.telegram.org/bot<api token>/sendMessage?chat_id=<chatID>&text=Is%20not,%20functional%20yet
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:02:01 --:--:--     0
curl: (35) Recv failure: Connection reset by peer

Running curl -4 ... and the job is done. api.telegram.org is resolved in ipv4 and ipv6, nc -v <api.telegram.org ipv6> 443 succeeded as well as a mtr to the IP.

I discovered that with another ipv6 address from the same server it's OK ! What's going on here ? The failing ipv6 address being from the IP range 2001:67c:1298::/48.

If someone had an idea ...

**Comment by Poolitzer** [comment]
Hi, we are not telegrams bot api, we are a wrapper for it in python. Since you have this issue with the API, maybe using [their bug tracker](https://github.com/tdlib/telegram-bot-api/issues) will help you.

**Comment by tootai** [comment]
OK, thanks for the information, will go there.
